documentation/modules/snip_permissions-info.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: SNIPPET
+
+If you are an addministrator, you can see and work with compoenents (providers, plans, etc.) for all projects.
+If you are a non-administrator, you can only see and work only with the compoenents of projects you have permissions for.
```

Comment:
right, that's one aspect - the other aspect that we wanted to cover here is the fine-grained permissions that the user needs to have. for instance, even when creating a new plan using predefined network/storage mappings, we (@yaacov) saw that the user still needs to be able to create a new network/storage mappings (because of the internal implementation of plan creation). that's why @qiyuann documented the permission they (QE) assigned to the user they tested with